### PR TITLE
Factor DB weights out into their own files

### PR DESCRIPTION
**Pull request by ggwpez**
Preparation for #10897:  
- Put the RocksDB and ParityDB weights into their own files.  

This will come in handy when switching to auto-generated files.

### CODE REVIEW

File: frame/support/src/lib.rs [hunk 1]
```diff
@@ -66,6 +66,8 @@ pub mod inherent;
 pub mod error;
 pub mod instances;
 pub mod migrations;
+mod paritydb_weights;
+mod rocksdb_weights;
```
Comment (bkchr) [inline frame/support/src/lib.rs:70]: They should be below the `weights` module.

Comment (ggwpez) [reply frame/support/src/lib.rs]: updated

Comment (ggwpez) [reply frame/support/src/lib.rs]: Looks like `fmt` is not happy with this: https://gitlab.parity.io/parity/substrate/-/jobs/1399806#L89  
I could add a newline, then it looks like this after `fmt`:  
```rust
...
pub mod traits;

mod paritydb_weights;
mod rocksdb_weights;
pub mod weights;
```

Comment (ggwpez) [reply frame/support/src/lib.rs]: Okay works now

**Review by bkchr — APPROVED** [review]

**Review by shawntabrizi — APPROVED** [review]

**Comment by ggwpez** [comment]
bot merge

**Pull request merged**